### PR TITLE
organize-tool 2.4.0

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  ==> /opt/homebrew/Cellar/organize-tool/2.4.0/bin/organize sim --config-file=/private/tmp/organize-tool-test-20220905-72552-eocvth/config.yaml
  Usage: organize sim [OPTIONS] [CONFIG]
  Try 'organize sim -h' for help.
  
  Error: No such option: --config-file
  
  Error: organize-tool: failed
  An exception occurred within a child process:
    Minitest::Assertion: Expected: 0
    Actual: 2
```

**Comment by Porkepix** [comment]
Fixed test, option to provide config file location was removed and is now looking in current directory.

**Comment by tfeldmann** [comment]
> Fixed test, option to provide config file location was removed and is now looking in current directory.

Yes, the `--config-file` option has been removed.
It does not search for a config file in the current directory, but in the default user folder.
The default path for the config can be shown with `organize reveal --path`.

To run a specific config file use it like this:

```sh
# Usage: organize sim [OPTIONS] [CONFIG]
organize sim path/to/your-config.yaml
# or
organize run conf.yaml
```

**Comment by chenrui333** [comment]
> Yes, the `--config-file` option has been removed.

from UX perspective, this should be brought back, which is more intuitive.

File: Formula/organize-tool.rb [hunk 1]
```diff
@@ -160,8 +120,8 @@ def install
 
     touch testpath/"homebrew.txt"
 
-    assert_match "Found: homebrew.txt", shell_output("#{bin}/organize sim --config-file=#{testpath}/config.yaml")
-    system bin/"organize", "run", "--config-file=#{testpath}/config.yaml"
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply Formula/organize-tool.rb:164]: that is exactly this, `system bin/"organize", "run", config_file` (config_file is variable defined above)

[the remark this answers is not in the document]
Comment (tfeldmann) [reply Formula/organize-tool.rb:164]: Looks good, but I don't have the permissions to accept the review

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**